ui/components/app/advanced-gas-popover/index.scss

```diff
@@ -0,0 +1,23 @@
+.advanced-gas-popover {
+  &__input-details-wrap {
+    display: flex;
+  }
+
+  & &__edit-link {
```

Comment:
Do you need this to be `.advanced-gas-popover .advanced-gas-popover__edit-link`, does `.advanced-gas-popover__edit-link` work the same way?

Reply:
Without that the styles for edit-link are not getting applied, it looks like below.
<img width="327" alt="Screen Shot 2021-11-19 at 9 59 23 AM" src="https://user-images.githubusercontent.com/43930900/142643667-0e922c3f-dac0-4e1a-b6e6-9a7f5bb22f08.png">

Reply:
fixed